cmake/OpenCVFindLAPACK.cmake

```diff
@@ -38,7 +38,7 @@ macro(ocv_lapack_check)
         set(_lapack_include_str_extern_C "extern \"C\" {\n")
         set(_lapack_include_str_extern_C_end "}\n")
     endif()
-    set(_lapack_include_str "${_lapack_include_str_extern_C}\#include \"${OPENCV_CBLAS_H_PATH_${_lapack_impl}}\"")
+    set(_lapack_include_str "${_lapack_include_str_extern_C}\#ifdef _MSC_VER\n\#include <complex.h>\n\#define lapack_complex_float _Fcomplex\n\#define lapack_complex_double _Dcomplex\n\#endif\n\#include \"${OPENCV_CBLAS_H_PATH_${_lapack_impl}}\"")
```

Comment:
Please split this line.
Escaping of `#` is not needed.
Example is [here](https://github.com/opencv/opencv/blob/3.4.6/cmake/OpenCVCompilerOptimizations.cmake#L785-L788).

---

It would be better to configure these defines via CMake with possibility of disabling them (if this would be fixed in OpenBLAS in the future).